pkgs/development/compilers/graalvm/community-edition/default.nix

```diff
@@ -32,54 +30,52 @@ in
   graalvm11-ce = mkGraal rec {
     config = {
       x86_64-darwin = {
+        inherit products;
         arch = "darwin-amd64";
-        products = commonProducts ++ [ "python-installable-svm" ];
       };
       x86_64-linux = {
+        inherit products;
         arch = "linux-amd64";
-        products = commonProducts ++ [ "python-installable-svm" ];
       };
       aarch64-darwin = {
+        inherit products;
         arch = "darwin-aarch64";
-        products = [
-          "graalvm-ce"
-          "native-image-installable-svm"
-        ];
         version = graalvm11-ce-dev-version;
       };
       aarch64-linux = {
+        inherit products;
         arch = "linux-aarch64";
-        products = commonProducts;
       };
     };
     defaultVersion = graalvm11-ce-release-version;
     javaVersion = "11";
     platforms = builtins.attrNames config;
   };
 
-  # TODO: fix aarch64-linux, failing during Native Image compilation
-  # "Caused by: java.io.IOException: Cannot run program
-  # "/nix/store/1q1mif7h3lgxdaxg6j39hli5azikrfla-gcc-wrapper-9.3.0/bin/gcc" (in
-  # directory"/tmp/SVM-4194439592488143713"): error=0, Failed to exec spawn
-  # helper: pid: 19865, exit value: 1"
   graalvm17-ce = mkGraal rec {
     config = {
       x86_64-darwin = {
+        inherit products;
         arch = "darwin-amd64";
-        products = commonProducts ++ [ "python-installable-svm" ];
       };
       x86_64-linux = {
+        inherit products;
         arch = "linux-amd64";
-        products = commonProducts ++ [ "python-installable-svm" ];
       };
       aarch64-darwin = {
+        inherit products;
         arch = "darwin-aarch64";
-        products = [
-          "graalvm-ce"
-          "native-image-installable-svm"
-        ];
         version = graalvm17-ce-dev-version;
       };
+      aarch64-linux = {
+        # TODO: error with native-image-installable-svm in aarch64-linux:
+        # "Caused by: java.io.IOException: Cannot run program
+        # "/nix/store/00000000000000000000000000000000-gcc-wrapper-9.3.0/bin/gcc" (in
+        # directory"/tmp/SVM-0000000000000000000"): error=0, Failed to exec spawn
+        # helper: pid: 19865, exit value: 1"
+        products = [ "graalvm-ce" ];
+        arch = "linux-aarch64";
+      };
```

Comment:
```suggestion
      aarch64-linux = {
        inherit products;
        arch = "linux-aarch64";
      };
```

Reply:
@thiagokokada This built in my vm successfully. Can we push these changes and double check with ofborg?

Don't forget to run the update-script to update the sources:smiley:.